pkg/blockbuilder/blockbuilder.go

```diff
@@ -74,10 +75,6 @@ func newWithSchedulerClient(
 	limits *validation.Overrides,
 	schedulerClient schedulerpb.SchedulerClient,
 ) (*BlockBuilder, error) {
-	if cfg.NoPartiallyConsumedRegion {
-		// We should not have a large buffer if we are putting all the records into a block.
-		cfg.ConsumeIntervalBuffer = 5 * time.Minute
-	}
```

Comment:
By removing this override, we can now try different buffer times when we have set NoPartiallyConsumedRegion to true